07-read-write-plot.Rmd

```diff
@@ -136,6 +136,35 @@ world2 = spData::world
 world3 = st_read(system.file("shapes/world.gpkg", package = "spData"))
 ```
 
+## Geographic web services
+
+In an effort to standardise web APIs for accessing spatial data, the Open Geospatial Consortium (OGC) has created a number of specifications for web services (collectively known as [OWS](http://www.opengeospatial.org/standards/owc)).
```

Comment:
Yes and yes, put in as a PR to get feedback. More broadly, is this new section useful/wanted? I think so but think it would need to be longer so more content welcome. In other words is this a useful code chunk and where next after demonstrating `GetCapabilities`?:

``` r
u = "http://geodata.nationaalgeoregister.nl/bag/wfs?"
g = "GetCapabilities"
res = httr::GET(url = u, query = list(request = g))
txt = httr::content(res, "text")
#> No encoding supplied: defaulting to UTF-8.
xml2::read_xml(txt)
#> {xml_document}
#> <WFS_Capabilities version="2.0.0" schemaLocation="http://www.opengis.net/wfs/2.0 http://schemas.opengis.net/wfs/2.0/wfs.xsd http://inspire.ec.europa.eu/schemas/inspire_dls/1.0 http://inspire.ec.europa.eu/schemas/inspire_dls/1.0/inspire_dls.xsd" updateSequence="1913" xmlns:wfs="http://www.opengis.net/wfs/2.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.opengis.net/wfs/2.0" xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:fes="http://www.opengis.net/fes/2.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:inspire_dls="http://inspire.ec.europa.eu/schemas/inspire_dls/1.0" xmlns:inspire_common="http://inspire.ec.europa.eu/schemas/common/1.0" xmlns:bag="http://bag.geonovum.nl">
#> [1] <ows:ServiceIdentification>\n  <ows:Title>BAG</ows:Title>\n  <ows:Ab ...
#> [2] <ows:ServiceProvider>\n  <ows:ProviderName>PDOK</ows:ProviderName>\n ...
#> [3] <ows:OperationsMetadata>\n  <ows:Operation name="GetCapabilities">\n ...
#> [4] <FeatureTypeList>\n  <FeatureType>\n    <Name>bag:ligplaats</Name>\n ...
#> [5] <fes:Filter_Capabilities>\n  <fes:Conformance>\n    <fes:Constraint  ...
```

<sup>Created on 2018-10-09 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

Reply:
@Robinlovelace truth to be told - I am not a regular user of these webservices, so I have no idea what should we show here.

Reply:
Know anyone else who is?